starter-code/views/auth/login.ejs

```diff
@@ -0,0 +1,19 @@
+<form action="/login" method="POST" id="form">
+  <h2>Login</h2>
+
+  <label for="username">Username</label>
```

Comment:
If you use the `for` attribute on `label` you need to give the corresponding `id` to the related input; or, you could put the `input` *inside* the label, like this: 

```html
<label>Username: <input type="text" name="username" placeholder="obiwan"></label>
```